code/src/components/AddTodos.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import todo from '../reducers/todo'
+
+
+const AddTodos = () => {
+  const [input, setInput] = useState('')
+
+  const dispatch = useDispatch()
+
+  const onAddTodo = () => {
+    dispatch(todo.actions.addTodo(input))
+    setInput('')
```

Comment:
Nice touch on setting the input value with empty quotes to clear the input field!